gaussian_blur_size: 7
gaussian_blur_sigma: 2
canny_edge_th: 120
hough_accum_resolution: 1
min_circle_dist: 40
hough_accum_th: 26.6
min_radius: 20
max_radius: 151
filter_range_min: 143
filter_range_max: 217
filter_debug: true